client/views/edit-function-definition.js

```diff
@@ -22,18 +22,20 @@ var modals = require('../modals');
 //views
 var View = require('ampersand-view');
 //templates
-var template = require('../templates/includes/editFunctionDefinition.pug');
+var editTemplate = require('../templates/includes/editFunctionDefinition.pug');
```

Comment:
Consider swapping to `let` since you're updating this line anyways

Reply:
This will be addressed in the code cleanup